test/Horde/Yaml/DumperTest.php

```diff
@@ -14,9 +14,12 @@
  * @package    Yaml
  * @subpackage UnitTests
  */
-class Horde_Yaml_DumperTest extends PHPUnit_Framework_TestCase
+
+namespace Horde\Yaml;
+
+class DumperTest extends \PHPUnit\Framework\TestCase
 {
-    public function setUp()
+    public function setUp() : void
```

Comment:
No space before the : please. See https://www.php-fig.org/psr/psr-12/ example at the bottom of chapter Overview.